NeMO/src/main/wdl/untar/UntarFiles.wdl

```diff
@@ -0,0 +1,56 @@
+version 1.0
+
+workflow UntarFiles {
+    input {
+        Array[File] tarball_lists
+        String project_name
+        String file_extension
+        String gsutil_output_path
+    }
+
+    scatter (tarball_list in tarball_lists) {
+        scatter (tarball in read_lines(tarball_list)) {
+            call ExtractFiles {
+                input:
+                    tarball = tarball,
+                    tarball_size = size(tarball, "GB"),
+                    project_name = project_name,
+                    file_extension = file_extension,
+                    gsutil_output_path = gsutil_output_path,
+            }
+        }
+    }
+}
+
+# assumes tarballs are not gzipped (no -z option for tar)
+task ExtractFiles {
+    input {
+        String tarball
+        Float tarball_size
+        String project_name
+        String file_extension
+        String gsutil_output_path
+    }
+
+    command <<<
+        set -euo pipefail
+
+        # untar the file (pipe using gsutil -u cat and ignore dir structure using --transform)
+        tar -vxf <(gsutil -u ~{project_name} cat ~{tarball}) --transform 's/.*\///g'
+
+         # copy to bucket
+         gsutil -m cp *~{file_extension} ~{gsutil_output_path}
+    >>>
+
+    runtime {
+        docker: "google/cloud-sdk:slim"
+        # if the input size is less than 1 GB adjust to min input size of 1 GB
+        disks: "local-disk " + ceil(1 * (if tarball_size < 1 then 1 else tarball_size)) + " HDD"
+        cpu: 1
+        memory: "3.5 GB"
```

Comment:
You can put `preemptible: 3` here to get cheaper disks.

Reply:
And tweak memory to be < 3.5 gig for papi v2.

Reply:
I have preemptible in the options json, should I put it hear as well?